qemu/tests/cfg/virtio_console.cfg

```diff
@@ -227,11 +231,14 @@
                         - boot_too_much_ports:
                             # arm doesn't use pci bus
                             no aarch64
-                            extra_params = " -device virtio-serial-pci,max_ports=32"
+                            extra_params_rhel = " -device virtio-serial-pci,max_ports=32"
+                            extra_params = " -device virtio-serial-pci,max_ports=512"
```

Comment:
I think you can use same param here, eg `extra_params = " -device virtio-serial-pci,max_ports=%s"`, then change the size accordingly.

Reply:
Thanks for advice. It's a good idea. I will update it.